src/components/InForm/InForm.tsx

```diff
@@ -103,7 +110,7 @@ export const InForm: React.FunctionComponent<any> = (props) => {
       }
       if (data.value !== "civ") {
         setFormData((f: INewInForm) => {
-          return { ...f, isNewCiv: false };
+          return { ...f, isNewCiv: "no" };
```

Comment:
Why change from true/false to "yes"/"no"?

Reply:
Switches supported boolean, but seems like RadioGroups do not